os/include/arpa/inet.h

```diff
@@ -171,6 +174,9 @@ uint32_t ntohl(uint32_t nl);
 /**
  * @brief Convert the unsigned short integer netshort from network byte order to host byte order
  *
+ * @details @b #include <arpa/inet.h>\n
+ * SYSTEM CALL API\n
```

Comment:
please check if this API is SYSTEM CALL or not.

Reply:
revised